Sources/Plasma/FeatureLib/pfPython/plPythonCallable.h

```diff
@@ -104,7 +104,11 @@ namespace plPython
             // Use Python's built-in vectorcall optimization for one argument.
             pyObjectRef arg = ConvertFrom(std::forward<Args>(args)...);
             plProfile_BeginTiming(PythonUpdate);
+#if PY_VERSION_HEX >= 0x03090000
             pyObjectRef result = _PyObject_CallOneArg(callable, arg.Get());
+#else
+            pyObjectRef result = PyObject_CallFunctionObjArgs(callable, arg.Get(), NULL);
```

Comment:
I haven't been able to find `_PyObject_CallOneArg` in the Python 3.8 headers like the commentary from the 3.9+ header indicate. It might be an API that is never actually publicly exposed at all in 3.8. Regardless, this fix is not correct because `PyObject_CallFunctionObjArgs` will allocate heap memory for the arguments. The whole point of this code is to use more optimized calling conventions present in modern Python versions that avoid unnecessary conversions and allocations. The correct fix is probably to remove this branch of the `if` statement and let single arguments be handed by the `else if constexpr (sizeof...(args) < 64)` block. Under the hood, `_PyObject_CallOneArg` does the same thing that this block does, so handling the two cases separately is probably not necessary.